app.js

```diff
@@ -245,7 +245,8 @@ document.addEventListener('DOMContentLoaded', () => {
   function gameOver() {
     if(current.some(index => squares[currentPosition + index].classList.contains('taken'))) {
       scoreDisplay.innerHTML = 'end'
-      clearInterval(timerId)
+      clearInterval(timerId);
+      document.removeEventListener("keyup",control);  //stops to click on any key
```

Comment:
for consistency with the rest of the code, you should remove the semi-colons ';' at the end of your lines.